src/ucp/core/ucp_mm.c

```diff
@@ -781,7 +787,7 @@ ucp_mpool_free(ucp_worker_h worker, ucs_mpool_t *mp, void *chunk)
     ucp_mem_desc_t *chunk_hdr;
 
     chunk_hdr = (ucp_mem_desc_t*)chunk - 1;
-    ucp_memh_put(worker->context, chunk_hdr->memh, 0);
+    ucp_memh_put(worker->context, chunk_hdr->memh, 1);
```

Comment:
why need to invalidate?
invalidate is only for error handling flows

Reply:
IMHO this action should revert action done at line 768 i.e. release memory. Without invalidating it will leave rcache entry in pgtable and will not release memory.

Reply:
i would expect that when we do the actual memory release (like munmap or shmdt), it will trigger UCM hooks to invalidate the rcache entry.

Reply:
also, this change seems not related to ucp_mem_map/unmap flow